torchvision/transforms/transforms.py

```diff
@@ -1059,10 +1059,14 @@ def get_params(brightness: Optional[List[float]],
         """Get the parameters for the randomized transform to be applied on image.
 
         Args:
-            brightness (tuple of float (min, max)): The range from which the brightness_factor is chosen uniformly.
-            contrast (tuple of float (min, max)): The range from which the contrast_factor is chosen uniformly.
-            saturation (tuple of float (min, max)): The range from which the saturation_factor is chosen uniformly.
-            hue (tuple of float (min, max)): The range from which the hue_factor is chosen uniformly.
+            brightness (tuple of float (min, max) or None): The range from which the brightness_factor is chosen
```

Comment:
Usually, we say `(tuple of float (min, max), optional)` instead of putting None. I have no strong opinion about that.

Reply:
Thanks for flagging! There are currently 4 different styles for pydocs, so I got confused. I corrected it and raised an issue to discuss the potential of standardizing them: #3003